index.html

```diff
@@ -1,28 +1,104 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
+  <head>
+    <meta charset="UTF-8" />
     <title>Grid Render</title>
     <style type="text/css">
-        html,
-        body {
-            margin: 0;
-            height: 100%;
-            background-color: #017DBD;
-            overflow: hidden;
+      html,
+      body {
+        margin: 0;
+        height: 100%;
+        background-color: #017dbd;
+        overflow: hidden;
+        font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+      }
+
+      .control-panel {
+        background-color: white;
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+        padding: 1rem;
+        border-bottom:5px ridge #020D78;
+
+        .controls {
+          display: flex;
+          justify-content: center;
+          margin-bottom: 1rem;
+
+          .preference {
+            font-size: 1rem;
+            display: flex;
+            flex-direction: column;
+            margin: 0 1rem;
+
+            input {
+                font-size: 1rem;
+                border-top: none;
+                border-left: none;
+                border-right: none;
+                border-bottom: 2px solid #020D78;
+            }
+          }
         }
+
+        .applybtn {
+            border: none;
+            height: 2rem;
+            width: 8rem;
```

Comment:
When different fonts are loaded the text can overflow.
![Screenshot from 2024-10-07 18-55-00](https://github.com/user-attachments/assets/55fed6f0-4e94-470f-a600-eb833caec1e6)

Please can you remove the fixed width, perhaps add some padding instead.